datanode/src/storage_interface.py

```diff
@@ -58,14 +53,47 @@
 USS_BASE_PREFIX = '/uss/gridcells/'
 TEST_BASE_PREFIX = '/test/'
 USS_METADATA_FILE = '/manifest'
-BAD_CHARACTER_CHECK = '\';(){}[]!@#$%^&*|"<>'
-CONNECTION_TIMEOUT = 2.5  # seconds
 DEFAULT_CONNECTION = 'localhost:2181'
 GRID_PATH = USS_BASE_PREFIX
 MAX_SAFE_INTEGER = 9007199254740991
 DELETE_ATTEMPTS = 3
 OK_STATUS = {200, 204}
 
+SQL_CREATE_TABLE = """
+  CREATE TABLE IF NOT EXISTS
+    grid (
+      tile STRING PRIMARY KEY NOT NULL,
+      uss_metadata STRING,
```

Comment:
do we want this to be a string, now that we have access to a relational db?

Reply:
I think, to begin with: yes. Going forward, I would expect the database layout to evolve, and the entities to be managed as first class citizens in a relational model.